modules/monitoring-resizing-a-persistent-storage-volume.adoc

```diff
@@ -0,0 +1,192 @@
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    prometheusK8s:
+      volumeClaimTemplate:
+        spec:
+          storageClassName: local-storage
+          resources:
+            requests:
+              storage: 100Gi
+----
++
+The following example changes a PVC that changes local persistent storage for Alertmanager to 40 gigabytes:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    alertmanagerMain:
+      volumeClaimTemplate:
+        spec:
+          storageClassName: *local-storage*
+          resources:
+            requests:
+              storage: 40Gi
+----
+
+** *To configure a PVC for a component that monitors user-defined projects*:
+.. Edit the `user-workload-monitoring-config` `ConfigMap` object in the `openshift-user-workload-monitoring` project:
++
+[source,terminal]
+----
+$ oc -n openshift-user-workload-monitoring edit configmap user-workload-monitoring-config
+----
+
+.. Update the PVC configuration for the monitoring component under `data/config.yaml`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: user-workload-monitoring-config
+  namespace: openshift-user-workload-monitoring
+data:
+  config.yaml: |
+    <component>:
+      volumeClaimTemplate:
+        spec:
+          storageClassName: <storage_class>
+          resources:
+            requests:
+              storage: <amount_of_storage> <1>
+----
+<1> Specify the new size for the storage volume.
++
+The following example changes the PVC size to 100 gigabytes the Prometheus instance that monitors user-defined projects:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: user-workload-monitoring-config
+  namespace: openshift-user-workload-monitoring
+data:
+  config.yaml: |
+    prometheus:
+      volumeClaimTemplate:
+        spec:
+          storageClassName: local-storage
+          resources:
+            requests:
+              storage: 100Gi
+----
++
+The following example changes the PVC size to 20 gigabytes for Thanos Ruler:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: user-workload-monitoring-config
+  namespace: openshift-user-workload-monitoring
+data:
+  config.yaml: |
+    thanosRuler:
+      volumeClaimTemplate:
+        spec:
+          storageClassName: *local-storage*
+          resources:
+            requests:
+              storage: 20Gi
+----
++
+[NOTE]
+====
+Storage requirements for the `thanosRuler` component depend on the number of rules that are evaluated and how many samples each rule generates.
+====
+
+. Save the file to apply the changes. The pods affected by the new configuration restart automatically.
```

Comment:
I don't think any pods will restart after saving the file, since in this procedure what we will see are new pods spawning once we patch the existing ones
```suggestion
. Save the file to apply the changes.
```

Reply:
This is not correct, the pods will indeed re-start

Reply:
Ok. Not making any changes here